docs/source/guide/shadows.md

```diff
@@ -0,0 +1,53 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Classical Shadows
+
```

Comment:
please add somewhere the two most important citations: one to the original classical shadow paper by preskill et al., and one to the robust classical shadow paper by flammia et al..

Reply:
thanks, I have added the citation in theory part, but will be added here as well.